test/e2e/storage/vsphere/vsphere_statefulsets.go

```diff
@@ -87,7 +87,7 @@ var _ = utils.SIGDescribe("vsphere statefulset", func() {
 		framework.ExpectNoError(e2esset.CheckMount(client, statefulset, mountPath))
 		ssPodsBeforeScaleDown := e2esset.GetPodList(client, statefulset)
 		gomega.Expect(ssPodsBeforeScaleDown.Items).NotTo(gomega.BeEmpty(), fmt.Sprintf("Unable to get list of Pods from the Statefulset: %v", statefulset.Name))
-		gomega.Expect(len(ssPodsBeforeScaleDown.Items) == int(replicas)).To(gomega.BeTrue(), "Number of Pods in the statefulset should match with number of replicas")
+		framework.ExpectEqual(len(ssPodsBeforeScaleDown.Items) == int(replicas), true, "Number of Pods in the statefulset should match with number of replicas")
```

Comment:
Can we change here like
`framework.ExpectEqual(len(ssPodsBeforeScaleDown.Items), int(replicas), "Number of Pods in the statefulset should match with number of replicas")` without `true`?

Reply:
ok,i get it